pyproject.toml

```diff
@@ -45,7 +45,7 @@ odmantic = { version = "*", optional = true }
 pydantic = { version = "*", optional = true, extras = ["email"] }
 typing-extensions = "*"
 attrs = {version = ">=22.2.0", optional = true}
-sqlalchemy = { version = ">=2", optional = true }
+sqlalchemy = { version = ">=1.4.49", optional = true }
```

Comment:
Is this constraint correct? `1.4.49` is the last version before v2.

Reply:
I imagine the implementation does not rely on the latest version so could be loosened however I have only tested this on this version. I imagine the upgrade path here for a user would be relatively straight forward.

If want to loosen, should CI install a different version?

Reply:
I think if it works under `1.4`, we can assume it works under the other versions (I think we should just test it out locally, but no need to include all the different versions in the CI). This is how it has been done for `pydantic` and there haven't been any issues so far.